packages/hackathon/src/scenes/ProjectsScene/components/utils.ts

```diff
@@ -24,4 +24,28 @@
 
  */
 
-export * from './ProjectView'
+import type { ITechnologyProps } from '../../../models'
```

Comment:
This file should be moved up a folder or two since its shared between projects and judging scene.